onnxruntime/contrib_ops/cuda/math/fft_ops_impl.h

```diff
@@ -0,0 +1,19 @@
+/*All contributions by Facebook :
+Copyright(c) 2016 Facebook Inc.
+==============================================================================*/
+/* Modifications Copyright (c) Microsoft. */
+
+#pragma once
+#include "core/providers/cuda/cuda_common.h"
+#include "fft_ops.h"
+
+namespace onnxruntime {
+namespace contrib {
+namespace cuda {
+
+template <typename T>
+void PostProcess(const std::vector<int64_t>& signal_dims, Tensor* Y, T* output_data);
```

Comment:
>Tensor* Y, T* output_data [](start = 58, length = 25)

this 2 parameters are dup, since you can get the output_data from Y. I think all you need is the output_data and the N -- output size.

Reply:
Done